chassis_test.go

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright (c) 2020 eBay Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+package goovn
+
+import (
+	"testing"
+)
+
+const (
+	CHASSIS_HOSTNAME = "fakehost"
+	CHASSIS_NAME     = "fakechassis"
+	ENCAP_TYPE       = "geneve"
+	IP               = "10.0.0.11"
+)
+
+func TestChassis(t *testing.T) {
+	db = getOVNDB()
+	if db == dbNB {
+		t.Skip("Skip running chassis test against nb db")
```

Comment:
In this case we should check if the API call returns proper error, e.g. "the API belongs to xx-DB but the client is connected to yy-DB"

Reply:
yes, we need to distinglish